src/main/java/com/minecolonies/colony/workorders/WorkOrderBuild.java

```diff
@@ -192,7 +192,7 @@ protected String getValue()
      */
     private boolean canBuildHut(int builderLevel, @NotNull CitizenData citizen, @NotNull Colony colony)
     {
-        return builderLevel >= upgradeLevel || builderLevel == 2
+        return builderLevel >= upgradeLevel || builderLevel == 5
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)